input/examples/encounter-consultation.xml

```diff
@@ -4,7 +4,7 @@
         <valueString value="Encounter - Consultation"/>
     </extension>
     <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-        <valueMarkdown value="Consultation with patient Fred Roberts"/>
+        <valueMarkdown value="This is a consultation encounter example using the *AU Core Encounter* profile.&#xa;&#xa;Patient: Fred Roberts"/>
```

Comment:
Don't linebreak Patient. Apply to all example markdown.

Reply:
Changed